clients/client-python/gravitino/api/expressions/Literals/literals.py

```diff
@@ -0,0 +1,138 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from decimal import Decimal
+from typing import Union
+from datetime import date, time, datetime
+
+from gravitino.api.expressions.literals.literal import Literal
+
+
+class LiteralImpl(Literal):
+    """Creates a literal with the given type value."""
+
+    _value: Union[int, float, str, datetime, time, date, bool, Decimal, None]
+    _data_type: (
+        str  # TODO: Need implement `api/src/main/java/org/apache/gravitino/rel/types`
+    )
+
+    def __init__(
+        self,
+        value: Union[int, float, str, datetime, time, date, bool, Decimal, None],
```

Comment:
Why do we have a `Decimal` here?

Reply:
Because we need to support `Decimal` type here.
- https://github.com/apache/gravitino/blob/main/api/src/main/java/org/apache/gravitino/rel/expressions/literals/Literals.java#L134

Reply:
Got it.